Tasks/NuGetCommand/nugetpublisher.ts

```diff
@@ -66,8 +67,13 @@ export async function run(nuGetPath: string): Promise<void> {
             }
         });
 
-        if (filesList && filesList.length < 1)
-        {
+        let searchPatternArray = searchPatternInput.split(";");
+        tl.debug(`Found ${filesList.length} files out of ${searchPatternArray.length}`);
```

Comment:
count of search patterns is not useful. Maybe show # of files and display the actual patterns used?